.github/workflows/github-deploy.yml

```diff
@@ -75,12 +74,8 @@ jobs:
           path: dist/*.tar.gz
 
   upload_pypi:
-    needs: [build_wheels, build_sdist]
+    needs: [build_wheels, build_sdist, check_manifest]
```

Comment:
GitHub branch protection was updated to only wait for this job.

I think that this should be ok. we will manually double-check the jobs before a release